hbase-server/pom.xml

```diff
@@ -437,24 +439,86 @@
           <skipAssembly>true</skipAssembly>
         </configuration>
       </plugin>
+      <plugin>
```

Comment:
I wonder if this could be done much better with the build-helper-maven-plugin, and just adding references to the directories instead of copying them.

Reply:
Sure let me switch to that

Reply:
done